src/content/docs/en/guides/troubleshooting.mdx

```diff
@@ -199,6 +199,30 @@ const answer = sum(2, 4);
 <Debug {answer} />
 ```
 
+## Creating minimal reproductions
+
+When you encounter a bug in Astro, it's most helpful to create a minimal reproduction of the issue. This helps us to understand the context, quickly identify the problem and fix it.
+
+A minimal reproduction should be a simplified version of the code that reliably demonstrates the issue.
+
+### Create a Stackblitz via [astro.new](https://astro.new).
```

Comment:
Nit: 'blitz' is capitalized in their branding, so I would change this for consistency. Also, heading with a dot.
```suggestion
### Create a StackBlitz via [astro.new](https://astro.new)
```

Reply:
Completely forgot that 🤦‍♂️! Thanks Yan!